ucm/sof-glkda7219max/sof-glkda7219max.conf

```diff
@@ -0,0 +1,32 @@
+SectionUseCase."HiFi" {
+	File "HiFi.conf"
+	Comment "Default"
+}
+
+SectionUseCase."Hdmi1" {
+	File "Hdmi1.conf"
+	Comment "Play to Hdmi1"
+}
+
+SectionUseCase."Hdmi2" {
+	File "Hdmi2.conf"
+	Comment "Play to Hdmi2"
+}
+
+SectionUseCase."Hdmi3" {
+	File "Hdmi3.conf"
```

Comment:
where are files Hdmi1,2,3?

Reply:
click into the commit, https://github.com/thesofproject/alsa-ucm-conf/pull/26/commits/0f42227ebc747f740187d55e7bec4ce2bed87a89, you should see these three files

Reply:
ok